src/sub-blocks/BasicCard/__stories__/BasicCard.stories.tsx

```diff
@@ -135,3 +171,19 @@ WithUrl.args = {
 WithContentList.args = {
     ...DefaultArgs,
 } as BasicCardProps;
+
+ControlPosition.argTypes = {
```

Comment:
is it necessary?

Reply:
We can omit controls' hiding, but I think that these controls are unnecessary.